Remotion/Web/Development.Analyzers/Web.Development.Analyzers.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net5.0</TargetFrameworks>
```

Comment:
wouldn't net48;net5.0 be 'correct' here? After all, we don't have a stand-alone netstandard2.0 target anywhere. But I'm not sure how the targetframework of analyzers relates to the rest of the stack.

(see also question in WebLibrary.props)